site/en/blog/permissions-chip/index.md

```diff
@@ -0,0 +1,142 @@
+  than automatically triggered by the site.
+- The permission is deemed essential and generally non-spammy. Currently this includes camera,
+  microphone, and camera paired with microphone.
+
+### Flow of the new design
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/amp4hx5iei2FvfXFt8gK.png", alt="ALT_TEXT_HERE", width="800", height="445" %}
+
+For non-essential permissions not triggered by a gesture, the prompt no longer intrudes on the site
+contents, and is not insistent on an immediate decision. The user can ignore the request chip until
+they have enough information to make a decision.
+
+With no interaction, after a short delay, the request chip will automatically collapse to just a
+blocked icon (to indicate the permission being temporarily blocked), before being dismissed
+entirely. The aim is to get out of the way of users who choose not to make a decision and can do so
+without any interaction.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/0xzCGLqbOFMcDGVvM1Z2.png", alt="ALT_TEXT_HERE", width="722", height="180" %}
+
+### Expected short-term impact
+
+In the short-term, until users get used to the new UI, it is likely that site owners will observe
+lower grant rates for sites, especially for those that auto-request permissions without priming or
+demanding a user gesture (a [bad practice](https://web.dev/notification-on-start/) anyway). This
+acknowledged downside is vastly outweighed by the less interruptive experience.
+
+### Best practices
+
+It is up to the site to ensure it provides the necessary context and only requests permissions at
+the appropriate and expected moment. Permissions that have been temporarily blocked—through a user
+ignoring the request or dismissing the prompt—can request the permission again within the same
+session. Only do this if the permission is essential for the site or feature to work, otherwise it
+runs the risk of annoying users and getting auto blocked. In those cases we show the
+[quiet messaging](https://blog.chromium.org/2020/01/introducing-quieter-permission-ui-for.html) that
+was introduced in Chrome 80. For more general guidance, see the article
+[Permission UX](https://developers.google.com/web/fundamentals/push-notifications/permission-ux).
+
+## Outlook and conclusions
+
+At the moment, the chip appears side by side next to the padlock, but the intention is to override
+the padlock in the future and add closer association of permissions and the site information
+surface. There are also plans on showing a confirmation with additional feedback on the decision
+made. The Chrome team is also investigating potentially more aggressive auto-blocking of permissions
+based on prior behavior. You will learn about the news here once these plans mature.
+
+Concluding, the new UI reduces the perceived insistence on a decision and improves the experience of
+browsing. Since most permission prompts are blocked or ignored, the met objective was to improve the
```

Comment:
```suggestion
Concluding, the new UI reduces the perceived insistence on a decision and improves the browsing
experience. Since most permission prompts are blocked or ignored, the met objective was to improve the
```